### ISSUE TITLE
[0.9.0.4][Server Issue] Players get disconnected after 10 minutes when they connect via the directory of servers

### ISSUE THREAD

**Issue by fillion07** [StrangeLoopGames/EcoIssues#18693]
Version 0.9.0.4
Server Issue

Players get disconnected after 10 minutes when they connect via the directory of servers. This behavior doesn't happen when they connect to the server via direct IP.

**Comment by SlayksWood** [comment]
@fillion07 Yhank you for the report. Can someone provide log file? or give please server name to test it.

**Comment by fillion07** [comment]
The server name is **TOKÉBEK ICITTE!!**

Where can I find my client logs? 

Will you need the server logs, if yes can I provide it to you privately?

**Comment by fillion07** [comment]
@SlayksWood Can we have an update on this? If you can tell what log file you need exactly (client or server) and where I can find it.  
I have try different configuration on the modem and firewall and I have no clue why it is still kicking players every 10 minutes when they are no connecting via direct IP.

**Comment by IcedForge** [comment]
Could you provide the server log and settings? 
You can send them via mail jens@strangeloopgames.com
 or discord Ice Forge #5886 if you want them kept private.

**Comment by fillion07** [comment]
Found out what was the problem:
I had my ports open in my router with port forwarding and had UPnP enabled. That was what created the quick of players after 10-15 minutes. If you decide to open ports in the router, you must disable UPnP otherwise it conflicts with port forwarding in the router. 

I also try to enable UPnP and not open ports in the router. I was able to connect, but it quicked me out of the game when my player appeared in the world. I reconnected again but failed while loading the world. I restarted the game (Eco client) and connected myself again to my game and I that point I was able to load the world and play for a good hour without disconnection. 

Conclusion I will for now just disabled UPnP a add ports for UDP and TCP in the ports forwarding section of my router.
